packages/block-editor/src/components/caption/index.native.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+const Caption = ( { accessible, accessibilityLabel, onBlur, onChange, onFocus, isParentSelected, isSelected, text } ) => {
+	// We still want to render the caption so that the soft keyboard is not forced to close on Android
+	const shouldCaptionDisplay = ! RichText.isEmpty( text ) > 0 || isParentSelected;
+
+	return (
+		<View style={ {	padding: 12, flex: 1, display: shouldCaptionDisplay ? 'flex' : 'none' } }
+			accessible={ accessible }
+			accessibilityLabel={ accessibilityLabel }
+			accessibilityRole={ 'button' }
+		>
+			<RichText
+				rootTagsToEliminate={ [ 'p' ] }
+				placeholder={ __( 'Write caption…' ) }
+				value={ text }
+				onChange={ onChange }
+				unstableOnFocus={ onFocus }
+				onBlur={ onBlur } // always assign onBlur as props
```

Comment:
Seems like `onBlur` is never passed to `Caption`

Reply:
Good eye! 🕵️‍♂️ I'll make this update. 

I'm curious, do you know why we "always assign onBlur as props"? I see this comment all over the place, but I never knew why. I'm guessing it might have to do with bubbling blur events back up from the native side?

Also, I was surprised that missing this didn't seem to cause any problems. I did a bit of digging and it looks like neither `image` nor `video` have an `onBlur` prop themselves, so it makes sense that passing nothing was no worse in this case than passing `undefined`.